### ISSUE TITLE
Any test results?

### ISSUE THREAD

**Issue by ken-ouyang** [Magicboomliu/Accelerator-Simple-Template#1]
Hello,

Thank you for making the training code available. I am interested in seeing how the model performs on scene flow datasets. Would it be possible for you to share any test results that you have conducted on the datasets?

**Comment by Magicboomliu** [comment]
I train the model with the initial weight of “stabilityai/stable-diffusion-2”, with a constant learning rate of 1e-5, and a `gradient_accumulation_steps` of 8, using two NVIDIA RTX 3090 with DDP, each card `batch_size`=1. 

I did not do the metrics evaluation, as far as the visualization results, it looks fine.  :)
* Input Image(screen short)
![image](https://github.com/Magicboomliu/Accelerator-Simple-Template/assets/36913896/72f37013-c412-4480-afff-5fff11e548c3)
* Estimated Depth using my trained model(screenshot by my phone, maybe a little slant)
![image](https://github.com/Magicboomliu/Accelerator-Simple-Template/assets/36913896/e048155f-3b0e-47a9-ae71-7a3ee6f2a9e7)  

Hope it helps.

**Comment by ken-ouyang** [comment]
Thanks for your reply. It looks pretty nice. I will definitely try it.

**Comment by Magicboomliu** [comment]
Hi~, Thank you for your appreciation. During Training, I strongly recommend you change the scheduler from `DDIM` to `DDPM` for better performance.    
Besides, I plan to release a new version to introduce the [ControlNet](https://arxiv.org/abs/2302.05543) into Marigold, which can further increase the robustness compared with the original paper in theory

**Comment by ZachL1** [comment]
@Magicboomliu Your excellent work has helped me a lot, thank you very much!

I would like to ask how to load the pre-training configuration from `stabilityai/stable-diffusion-2` after using DDPM instead of DDIM? I noticed that there are only DDIM config files in `stabilityai/stable-diffusion-2/scheduler`. Do you use the DDIM configuration file directly in your implementation? like this:
```python
noise_scheduler = DDPMScheduler.from_pretrained('stabilityai/stable-diffusion-2', subfolder='scheduler')
```
